src/containers/org/members/MembersSection.js

```diff
@@ -145,34 +145,31 @@ const MembersSection = ({
   const filteredMembersCount = filteredMembers.count();
   const pageMembers = filteredMembers.slice(paginationIndex, paginationIndex + MAX_PER_PAGE);
 
+  const disableAddMember = selectedRole ? false : !isOwner;
+
+  const handleAddMemberOnClick = () => {
+    if (selectedRole) {
+      setIsVisibleAssignRoleModal(true);
+    }
+    else {
+      setIsVisibleAddMemberToOrgModal(true);
+    }
```

Comment:
Just to be super defensive, I would explicitly set the visibility of the other modal to false.

Reply:
unless i set up a reducer to do both setStates at once, there is a chance it would incur extra re-renders due to multiple updates. The appropriate modal is completely unmounted when selectedRole is falsy anyway. Leaving as is until there's a tangible reason to change.